bonsai-core/src/test/scala/com/stripe/bonsai/BitsetSpec.scala

```diff
@@ -18,7 +18,7 @@ class IndexedBitSetSpec extends WordSpec with Matchers with Checkers {
 
   implicit val arbitraryBitSetWithIndex: Arbitrary[BitSetWithIndex] =
     Arbitrary(for {
-      len <- arbitrary[Short].map(_.toInt)
+      len <- arbitrary[Short].map(_.toInt) if len >= 0
```

Comment:
yeah, I'm an idiot. Sorry for the goof up. It should be uniformly distributed if the input is uniform.